test/configuration_unittest.cc

```diff
@@ -0,0 +1,61 @@
+#include "../src/configuration.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+void TestDefaultConfig(google::MetadataAgentConfiguration& config) {
```

Comment:
I'm not too happy about this, as it'll need to be updated when defaults change, but I don't have any better ideas for now, so let's keep it.

Reply:
Agreed